### DISCUSSION TITLE
Email queue not receiving email

### DISCUSSION THREAD

**Discussion by guxtavo** [Peppermint-Lab/peppermint#321] [Q&A]
How can I configure logs for the email queues so I can see why I'm not receiving anything? 
I see this in docker logs:

loop completed
No new messages

Is it checking the correct INBOX (+INBOX)? I'm able to use the email account in mutt and if I create a ticket in the dashboard I can receive emails as well, but nothing that gets in the INBOX is recognized by peppermint, help!

(Rethoric)  Why there is a check for the SMTP but no check for the IMAP?

**Comment by potts99** [comment]
will create a email logger and have this in the admin panel @guxtavo

**Comment by thephox1982** [comment]
I just got all setup with Peppermint and have my pop/imap info in and been waiting 20 minutes now for it to get the test email I sent to my email account, still nothing. I was hoping this was the software to escape the expensive services but seems to be another dead end. :(

I looked to see if there was a timer interval setting or configuration or even manual refresh and nothing.
